# csm-2024

This repository contains more details for the bacterial β-glucuronidase diversity in the Human gut - 73rd Annual Conference of the Canadian Society of Microbiologists poster presented by A.R Raphenya.

# Abstract Title:

Exploring bacterial β-glucuronidase diversity in the Human gut

# Author(s) Information:

Amogelang R. Raphenya, Akash Mehta, Nawal Masood, Michaela Hughes-Butler, Mugdha Dave, Mahrukh Khan, Tiffany E. Ta, Michael G. Surette, Gerard D. Wright, Andrew G. McArthur

# Abstract:

The human gastrointestinal tract (gut) has bacteria that contribute to the metabolism of administered drugs. The gut microbiome also converts drug metabolites, which include drug conjugates from the liver. Bacterial β-glucuronidase genes encode enzymes that convert most parent drug metabolites or prodrugs and are prevalent in the human gut microbiomes we analyzed.

Currently, there are a few resources that systematically catalog all genes involved in gut microbiome drug metabolism. Yet, there is a need to understand drug metabolism as it can reduce time and resources during the drug development process by avoiding adverse reactions or treatment failure through the gut microbiome. 

We developed an ontology-centric database to catalog all reported drug-metabolizing genes systematically, termed the Human Microbiome Drug Metabolism (HMDM) database. Currently, 50 bacterial genes and 42 gene variants to the β-glucuronidase (uidA gene) are curated. We developed a prevalence module to assess the frequency of the drug-metabolizing genes from the species commonly found in the human gut microbiome. The gut microbiome genomic data obtained from the National Center for Biotechnology Information (NCBI) Datasets were analyzed using a newly developed software called the Drug Metabolising Enzyme (DME). The DME predicts potential drug-metabolizing genes based on curated genes in the HMDM database.

The β-glucuronidases are abundant in the human gut and metabolize different substrates, including carbohydrates, glucuronides, and glycosides. Yet, a few have been characterized with specific drugs they hydrolise. A study by Pollet and colleagues identified key loops in the β-glucuronidase enzymes. In our study, we identified other β-glucuronidase homologs with different residues in the identified loops. We used sequence similarity networks (SSNs) to compare the homologs and modeled each enzyme using ColabFold to predict the structure. Our study aims to identify their diversity and substrate preference which will help to guide biochemical tests on these homologs. This will help in identifying drugs that β-glucuronidase enzymes might metabolize.

# Resource

- https://www.brightcarbon.com/brightslide/
- https://www.brightcarbon.com/blog/effective-academic-posters-powerpoint/

